recipes/sqlpp11/all/conanfile.py

```diff
@@ -26,3 +26,8 @@ def package(self):
 
     def package_id(self):
         self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "Sqlpp11"
+        self.cpp_info.names["cmake_find_package_multi"] = "Sqlpp11"
```

Comment:
```suggestion
        self.cpp_info.filenames["cmake_find_package"] = "Sqlpp11"
        self.cpp_info.filenames["cmake_find_package_multi"] = "Sqlpp11"
        self.cpp_info.names["cmake_find_package"] = "sqlpp11"
        self.cpp_info.names["cmake_find_package_multi"] = "sqlpp11"
```

It can be fixed now.

config file: https://github.com/rbock/sqlpp11/blob/54cf48b7edda1319a7c01ae55ea0b78dfd5f6c64/CMakeLists.txt#L110
target: https://github.com/rbock/sqlpp11/blob/54cf48b7edda1319a7c01ae55ea0b78dfd5f6c64/CMakeLists.txt#L104